### PR TITLE
update WMI dict

### PR DESCRIPTION
**Pull request by ghilesmeddour**

### CODE REVIEW

**Comment by idlesign** [comment]
Thank you. I've reconfigure CI to use GitHub Actions. Please rebase your pull requests.
By the way #12 ruined the tests and those need to be updated before we can proceed.

File: vininfo/dicts/wmi.py [hunk 1]
```diff
@@ -8,36 +8,74 @@
...
     '1C': 'Chrysler',
+    '1C3': 'Chrysler',
     '1C4': 'Dodge',
     '1C6': 'Ram',
     '1C8': 'Chrysler',
+    '1C9': 'Chance',
     '1CN': 'Lincoln',
+    '1CY': 'Crane Carrier',
     '1D': 'Dodge',
+    '1D3': 'Dodge',
```
Comment (idlesign) [inline vininfo/dicts/wmi.py:51]: Two letter dict keys already decribe implicitly three letters varians. E.g. 1D includes 1D3, 1D4 etc.
That's done to keep the dict minimized. So those need to be dropped.
If unsure you can always check corner cases in tests.

Comment (ghilesmeddour) [reply vininfo/dicts/wmi.py:51]: I agree, but I wonder if it's not a good idea to remove all the two-character keys? Because in reality the manufacturer is always identified by three characters, here the two character key are used as an easy fallback but in reality this will not give a correct in all cases.

```
    @property
    def wmi(self) -> str:
        """WMI (World Manufacturers Identification)"""
        return self.num[:3]

    @property
    def brand(self) -> Brand:
        """Brand object."""

        wmi = self.wmi

        brand = WMI.get(wmi)

        if not brand:
            brand = WMI.get(wmi[:2])
 ```

Comment (idlesign) [reply vininfo/dicts/wmi.py:51]: > but in reality this will not give a correct in all cases.

Let's say it is considered correct till it's not proved otherwise %P
So let's keep a compacted version until we have issues reported.
